src/style.css

```
@@ -0,0 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

don't reset `margin padding` in all tags, it's very difficult process 
```suggestion
* {
  box-sizing: border-box;
}
```
```
h1, h2, p, ... {
  margin: 0;
  padding: 0;
}
```